.github/styles/kong/Terms.yml

```diff
@@ -0,0 +1,8 @@
+extends: substitution
+message: Use '%s' instead of '%s'.
+level: error
+ignorecase: true
+swap:
+  kong: Kong
+  k8s: Kubernetes
+  
```

Comment:
Should we extend this to capitalise all products (e.g. Konnect)?

Even better, should we prevent direct usage of Konnect and suggest the config variable for that product?